### PR TITLE
push to cgr.dev

### PR DESCRIPTION
**Pull request by imjasonh**
This will need a change to `setup-chainctl` to correctly configure creds for `cgr.dev`, but once that's in we should be good to go.

cc @mattmoor

### CODE REVIEW

**Comment by dlorenc** [comment]
Will we also be able to remove the random sleeps?

**Comment by jdolitsky** [comment]
I'm looking into the CI failures

File: .github/actions/release-image/action.yml [hunk 1]
```diff
@@ -296,6 +296,9 @@ runs:
       if: inputs.chainguardIdentity != ''
       with:
         identity: ${{ inputs.chainguardIdentity }}
+    - run: |
```
Comment (jdolitsky) [inline .github/actions/release-image/action.yml:299]: `if: inputs.chainguardIdentity != ''`

Comment (mattmoor) [inline .github/actions/release-image/action.yml:299]: We want the `if` check on the identity (or drop it entirely)

**Comment by mattmoor** [comment]
@dlorenc re: sleeps, we need to reconfigure the workflow to push to `cgr.dev` during the main part of the workflow (and copy to ghcr.io/chainguard-images at the end), and then we should be able to 🤞

**Review by mattmoor — APPROVED** [review]

**Review by jdolitsky — APPROVED** [review]
@imjasonh merge at will

**Pull request merged**